index.js

```diff
@@ -86,7 +86,7 @@ function updateLock(file, options) {
     const lock = locks[file];
 
     /* istanbul ignore next */
-    if (lock.updateTimeout) {
+    if (lock.updateTimeout != null) {
```

Comment:
It will return false if `lock.updateTimeout == 0`. I'm not entirely sure if the integer returned by `setTimeout` will always return a non-zero value, that's why I added that part.